src/utils/pick/pick.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+export default function pick (obj: Object, picks: string []): Object {
+  const picked = {}
```

Comment:
Favor `.reduce` for data composition. Intermediate mutable state is, well, moderately okay, if kept locally, but teaching yourself to think with immutability is one of the best things which may happen to a programmer.

See the example of the `pick` function above.